test/integration/backward_compatible/parallel/sql/ExecuteTest.js

```diff
@@ -639,7 +647,7 @@ describe('SqlExecuteTest', function () {
                 </hazelcast>
             `;
 
-            cluster = await RC.createCluster(null, LITE_MEMBER_CONFIG);
+            const cluster = await testFactory.createClusterForParallelTests(null, LITE_MEMBER_CONFIG);
```

Comment:
Why don't have this and the new test you added below. They look very similar (`should return an error if sql query sent to lite member`)

Reply:
This should be a merging error let me delete one of them.